sky/templates/kubernetes-ray.yml.j2

```diff
@@ -46,10 +42,12 @@ provider:
             labels:
                 parent: skypilot
             name: autoscaler
+        # TODO(romilb): This is a very permissive role - gives all access in the
+        #  namespace. We should restrict this.
```

Comment:
Should we mention that this is for the `autodown` to work, so in the future we know how to test the restricted permissions?